src/cursor/cur_file.c

```diff
@@ -249,6 +260,8 @@ __curfile_insert(WT_CURSOR *cursor)
 	    (!F_ISSET(cursor, WT_CURSTD_APPEND) &&
 	    F_MASK(cursor, WT_CURSTD_KEY_SET) == 0));
 
+	__wt_stat_write_op_histogram(session, WT_TIMEDIFF_US(stop, start));
```

Comment:
like at other places, lets update the statistics just after we save end time in the stop variable.